ui/package.json

```diff
@@ -17,6 +17,7 @@
     "panels-plugin"
   ],
   "devDependencies": {
+    "@formatjs/intl-numberformat": "^7.4.1",
```

Comment:
Could potentially remove this dependency since I only ended up using one method that validates against [unit identifier standard](https://tc39.es/proposal-unified-intl-numberformat/section6/locales-currencies-tz_proposed_out.html#sec-issanctionedsimpleunitidentifier), but there may be other [future use-cases for it](https://github.com/formatjs/formatjs)

Reply:
I'm cool with keeping it, but is this a root dev dependency? Feels like this actually belongs as a dependency of a specific package?

Reply:
Whoops, just moved to panels-plugin's package.json!